nlp/README.md

```diff
@@ -0,0 +1,22 @@
+## DACCS - NLP docker image
+
+This image is able tu run all the python files required for the nlp notbooks.
```

Comment:
it would be more accurate with "this image contains all the nlp notebook dependencies"